packages/react-components/src/components/abstract-xy-output-component.tsx

```diff
@@ -751,7 +770,7 @@ export abstract class AbstractXYOutputComponent<
             rightPos = xLocation;
         }
 
-        const tooltipData = {
+        const tooltipData: TooltipData = {
             title: timeLabel,
             dataPoints: points,
             top: topPos,
```

Comment:
The four positioning fields are not used, so the position calculations above are also not needed, and the x,y function parameters are not needed.

Reply:
Thanks.  I went ahead and removed the function inputs `(x, y)` as well as the following keys from `TooltipData`: `top, bottom, right, left` and `opacity, transition`.

Function calls of `tooltip` have also been fixed in `xy-output-component` and `xy-output-overview-component`.